### DISCUSSION TITLE
Fields operations on select

### DISCUSSION THREAD

**Discussion by codesian** [PostgREST/postgrest#3390] [Q&A] [answered]
Its there a way to concat or run functions on fields on the fly, with a function or operator?

table?select=value:id,text:concat(name+desc)
table?select=value:id,text:name||desc
table?select=name||desc

Thanks!

**Comment by wolfgangwalther** [comment, marked as answer]
No there is not. You can created [computed fields](https://postgrest.org/en/v12/references/api/computed_fields.html) to do that.
